modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,75 @@
+# Candidate Parameters
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data entry personnel
+2. Site coordinators who are validating the data
+
+## Scope
+
+The Candidate Parameters module stores all information 
+for a candidate that are not timepoint dependent. All data 
+collected during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions, either one
+is sufficient to access the module.
+
+ - `candidate_parameter_view`
+    - This permission gives the user access to view the module and all of its tabs.
+
+ - `candidate_parameter_edit`
+    - This permission gives the user access to edit the module and all of its tabs.
+
+## Configurations
+
+The following configurations affect the usage of the Candidate Parameters module:
+
+ - `useProband` 
+    - This setting displays/hides the Proband tab of the module.
+
+ - `useFamilyID`
+    - This setting displays/hides the Family tab of the module.
+
+ - `useConsent`
+    - This setting displays/hides the _Consent Status_ tab of the module.
+
+The _Consent Status_ tab dynamically displays consents defined under the 
+`<ConsentModule> <Consent>` tags in the config.xml file. 
+
+ - The `<name> </name>` tag defines the field name on the form and is required to 
+ match the name of the sql fields in the `participant_status` and 
+ `consent_info_history` tables of the database. 
+ - The `<label> </label>` tag defines the front-end label displayed in the candidate 
+ parameters module for the consent in question. 
+ 
+The _Candidate Information_ tab dynamically displays customized fields configured in 
+the database. The following SQL tables affect the behaviour of these fields:
+
+ - `parameter_type_category`
+    - Identifies the categories for `parameter_type` entries. This table requires an 
+    entry for the _Candidate Parameters_ module for customizations to take effect.
+ - `parameter_type`
+    - Lists fields to be displayed on the front-end of the module and all their 
+    attributes.
+ - `parameter_type_category_rel`
+    - Relational table associating the Candidate Parameters from the `parameter_type` 
+    table to the entry in the `parameter_type_category` 
+ - `parameter_candidate`
+    - Table storing values saved for each custom field added in the 3 tables above.
```

Comment:
The format of these 4 points should be consistent. At the moment, the first two say what the table does, and the last two say what table it is. It should either all start with verbs, or all start with nouns, if ya catch my drift

Reply:
product of getting interrupted 98.5 times whicle editing a file